jobs/repos_without_about_yml.rb

```diff
@@ -17,4 +17,10 @@
   end
 
   send_event('repos_without_about_yml', unordered: true, items: items)
+
+  total_org_repos = ENV['GITHUB_ORG_REPOS'].split(',').size.to_f
+  repos_without_about = repos_without_about_yml.size.to_f
+  percentage_adoption = (1 - repos_without_about / total_org_repos) * 100
+
+  send_event('about_percentage', value: percentage_adoption.round(2))
```

Comment:
I would if I could figure out how to. I haven't been able to find any examples in the wild of anyone testing what gets sent to a widget. I asked on the Dashing repo and never got a response: https://github.com/Shopify/dashing/issues/615